cli/src/commands/group.rs

```diff
@@ -29,11 +33,22 @@ pub async fn handle_group(api: &PhylumApi, matches: &ArgMatches) -> CommandResul
 }
 
 /// Handle `phylum group create` subcommand.
-pub async fn handle_group_create(api: &PhylumApi, matches: &ArgMatches) -> CommandResult {
+pub async fn handle_group_create(
+    api: &PhylumApi,
+    matches: &ArgMatches,
+    config: Config,
+) -> CommandResult {
     let group_name = matches.get_one::<String>("group_name").unwrap();
-    match api.create_group(group_name).await {
-        Ok(response) => {
-            print_user_success!("Successfully created group {}", response.group_name);
+
+    let response = if let Some(org) = config.org() {
+        api.org_create_group(org, group_name).await
+    } else {
+        api.create_group(group_name).await.map(|_| ())
+    };
+
+    match response {
+        Ok(_) => {
+            print_user_success!("Successfully created group {}", group_name);
             Ok(ExitCode::Ok)
         },
         Err(PhylumApiError::Response(ResponseError { code: StatusCode::CONFLICT, .. })) => {
```

Comment:
I still think it would be a nicer UX to make it clear that org groups can't be created if you are not the org admin. The current output in that case is not great for users:

```
❯ target/debug/phylum -c ~/.config/phylum/staging.settings.yaml group create --org phylum delme_org_group_create_test
❗ Error: HTTP request error (403 Forbidden):

access forbidden, insufficient privileges, re-authentication will not help
```

Is a change required in API or could this be improved here?

Reply:
Org groups are currently always accessible to all members as far as I know. So I think in the scenario where this returns a 403 we can be certain that the issue is due to not being an admin.

Reply:
@maxrake Can you open an API ticket for this?

Even if CLI produces a better message, the API response is still lacking.